test.sh

```diff
@@ -87,9 +87,13 @@ for version in "${meteor_versions[@]}"; do
 	elif [[ "${version}" == 2.3.6 ]] || [[ "${version}" == 2.4 ]]; then
 		node_version='14.17.6'
 
-	# Versions >= 2.5 need Node 14.18.1
-	else
+	# Version 2.5 needs Node 14.18.1
+	elif [[ "${version}" == 2.5 ]]; then
 		node_version='14.18.1'
+
```

Comment:
Please add a comment noting which versions are skipped and therefore unsupported.

Reply:
Thanks! https://github.com/disney/meteor-base/pull/100/commits/8c0306f2ed5152a448609e854ee5482481b2bd88